js/sign.js

```diff
@@ -0,0 +1,52 @@
+const errorMessageEmail = document.querySelector('.enter-email');
+const errorMessageEmail2 = document.querySelector('.wrong-email');
+const emailInput = document.querySelector('.email-input');
+
+emailInput.onblur = function (e) {
+  const regex = new RegExp('[a-z0-9]+@[a-z]+\.[a-z]{2,3}');
+  if (emailInput.value === ''){
+    errorMessageEmail.setAttribute('class', 'display-span')
+    errorMessageEmail2.removeAttribute('class');
+    errorMessageEmail2.setAttribute('class', 'display-none'); 
+    emailInput.setAttribute('id', 'border-color')
+  } else if (emailInput.value.match(regex) === null){
+    errorMessageEmail.removeAttribute('class');
+    errorMessageEmail.setAttribute('class', 'display-none');
+    errorMessageEmail2.setAttribute('class', 'display-span')
+    emailInput.setAttribute('id', 'border-color')
+  } else {
+    errorMessageEmail.removeAttribute('class');
+    errorMessageEmail2.removeAttribute('class');
+    errorMessageEmail.setAttribute('class', 'display-none');
+    errorMessageEmail2.setAttribute('class', 'display-none');
+    emailInput.removeAttribute('id'); 
+  }
```

Comment:
else-if, else 보다는 요렇게 조건식을 다뤄주면 괜찮을 것 같아요!

```suggestion
  if (emailInput.value === ''){
    errorMessageEmail.setAttribute('class', 'display-span')
    errorMessageEmail2.removeAttribute('class');
    errorMessageEmail2.setAttribute('class', 'display-none'); 
    emailInput.setAttribute('id', 'border-color')
    return;
  } 
  
  if (emailInput.value.match(regex) === null){
    errorMessageEmail.removeAttribute('class');
    errorMessageEmail.setAttribute('class', 'display-none');
    errorMessageEmail2.setAttribute('class', 'display-span')
    emailInput.setAttribute('id', 'border-color')
    return;
  } 
  
  errorMessageEmail.removeAttribute('class');
  errorMessageEmail2.removeAttribute('class');
  errorMessageEmail.setAttribute('class', 'display-none');
  errorMessageEmail2.setAttribute('class', 'display-none');
  emailInput.removeAttribute('id'); 
  
```